joplin/settings.py

```diff
@@ -215,6 +215,7 @@
 DEBUG_TOOLBAR = bool(strtobool(os.environ.get('DEBUG_TOOLBAR', str(False))))
 MONITOR_PERFORMANCE = bool(strtobool(os.environ.get('MONITOR_PERFORMANCE', str(False))))
 
+WAGTAILEMBEDS_RESPONSIVE_HTML = True
```

Comment:
maintains previous compatibility, we still use these classes in janis (or at least they _are_ in janis)

Reply:
Gotchya!👍
... also, you responded to both of these questions in like 10 seconds.